src/Features/CSharp/Portable/DocumentationComments/DocCommentConverter.cs

```diff
@@ -45,14 +47,14 @@ public override SyntaxNode Visit(SyntaxNode node)
             // Check the leading trivia for doc comments.
             if (node.GetLeadingTrivia().Any(SyntaxKind.SingleLineDocumentationCommentTrivia))
             {
-                var newLeadingTrivia = new List<SyntaxTrivia>();
+                using var _ = ArrayBuilder<SyntaxTrivia>.GetInstance(out var newLeadingTrivia);
 
-                foreach (var trivia in node.GetLeadingTrivia())
+                foreach (var trivia in node.GetLeadingTrivia().SkipWhile(t => !t.IsKind(SyntaxKind.SingleLineDocumentationCommentTrivia)))
```

Comment:
this was a required and unpleasant change to make.  it has to do with how the formatting engine subtly treats the source produced by ILSpy different from normal MAS source.  Specifically in MAS we generate like this:

```
int Foo {get;}//
// this is the comment for Bar
int Bar {get;}
```

With our formatting rule the formatter converts this to the expected:

```
int Foo {get;}
//
// this is the comment for Bar
int Bar {get;}
```

However, with decompilation we start with:

```
int Foo {get;}

//
// this is the comment for Bar
int Bar {get;}
```

And no matter what rules i specify the formatting engine will not remove that blank line.  I've debugged through the deep bowels of teh formatting engine, and i cannot make any sort of sense of what it is doing here.  it literally has an embedded rule deep in the core engine that simply overries any actual provided rules here and change things to be 'preserve lines' for comments.  I do not want to change that code, so i'm taking hte much simple approach here of just not having the blank line trivia at the start be jammed in.  This seems to have no negative consequences anywhere, and is trivial to do.